### ISSUE TITLE
tflint should return non-zero exit code if command fails

### ISSUE THREAD

**Issue by pcgeek86** [terraform-linters/tflint#1613]
### Summary

tflint does not return a non-zero exit code if the command line arguments are incorrectly specified. 
Notice the extra space character, after the colon, in the command below.

```
C:\git\terraform-gcp-containers> tflint /disable-rule:terraform_required_providers /disable-rule: terraform_required_version
Failed to parse CLI arguments; Failed to load `terraform_required_version`: File not found
C:\git\terraform-gcp-containers> $LASTEXITCODE
0
```

### Command

tflint /disable-rule:terraform_required_providers /disable-rule: terraform_required_version

### Terraform Configuration

```terraform
n/a
```

### TFLint Configuration

```HCL
n/a
```

### Output

```Shell
Failed to parse CLI arguments; Failed to load `terraform_required_version`: File not found
```

### TFLint Version

TFLint version 0.43.0

### Terraform Version

n/a

### Operating System

- [ ] Linux
- [ ] macOS
- [X] Windows
